app/models/data-operation.ts

```diff
@@ -289,6 +289,8 @@ export default class DataOperation {
 
   ignoreErrors?: boolean;
 
+  destinationUrl: string;
```

Comment:
We should remove this line. There is already a stagingLocation in the data operation model. We should set the stagingLocation as appropriate if there is a destinationUrl for the job.